content/learn/beginner/control-flow-1.md

```diff
@@ -0,0 +1,213 @@
++++
+title = "Control Flow 1"
+weight = 230
++++
+
+Until now, we have just written some linear code that executes from top to bottom. However, in real-world applications,
+we often need to make decisions based on certain conditions. This is where *control flow* comes in.
+
+You already have an intuitive understanding of control flow. For instance, when you cross the street, you look left and
+right to see if any cars are coming. If there are no cars, you cross the street. If there are cars, you wait until they
+have passed, then check again. This is a simple example of a decision-making process.
+
+## Comparison Actions
+
+In programming, we can make decisions by comparing values to each other. We can compare numbers, strings, and other data
```

Comment:
might be worth mentioning for programmers with prior experience that comparison is done via functions, not binary operators